Dockerfile

```diff
@@ -0,0 +1,55 @@
+# Convenience Dockerfile to make it easy to replicate the Travis build locally.
+# Build this using `docker build`, then run it to get a shell.
+#
+# From the container shell, run the `updateRepo.sh` script. This will execute
+# the project update process, but will not commit the changes. You can inspect
+# what the script has done by looking in the /project/current/<branch name> dirs.
+#
+FROM swift:5.0.2
+
+# Fix python directories in base image
+RUN if [ -d "/usr/lib/python2.7/site-packages" ]; then mv /usr/lib/python2.7/site-packages/* /usr/lib/python2.7/dist-packages && rmdir /usr/lib/python2.7/site-packages && ln -s dist-packages /usr/lib/python2.7/site-packages ; fi
+
+# Install basic dependencies
+RUN apt-get install -y curl
```

Comment:
Each `RUN` statement creates a new layer in the Docker image. It's best practice to minimise layers so we should install all the necessary packages in one `RUN` statement.

```dockerfile
RUN apt-get update && apt-get install -y curl libssl-dev libcurl4-openssl-dev vim nodejs pandoc rsync
```

Reply:
Done in https://github.com/IBM-Swift/generator-swiftserver-projects/pull/28/commits/aef6e2e08e69009204bd75d5d155f61439cd3b5a

I made it multi-stage so that the 'test project generation' stage doesn't add to the final image (as the final task of running updateRepo.sh doesn't make use of the project that was generated by generateProject.sh)

Reply:
Nice.